### PR TITLE
update Radon page

### PR DESCRIPTION
**Pull request by JedrickZablan**
I have added information about the element Radon to our collaborative repository. Thank you to my peers for all the help.

### CODE REVIEW

File: elements/Radon.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Radon
+---
+
+Symbol: Rn
+Atomic number: 86
+
+Radon is a radioactive element that naturally occurs as a gas. The atomic mass of Radon is 222 Da. Radon is used in cell damage and cancer treatment. However, radon levels are regularly tested in homes to ensure because it can seep through well water.
```
Comment (kassadym) [inline elements/Radon.md:9]: Three sentences, good job

**Review by kassadym — APPROVED** [review]

**Review by kassadym — APPROVED** [review]
Looks good! Ready to merge :)

**Comment by JedrickZablan** [comment]
closes #126

**Comment by JedrickZablan** [comment]
addresses #126

**Pull request merged**